### ISSUE TITLE
Interact v1.3.0-alpha.1 node module  giving error with UglifyJs

### ISSUE THREAD

**Issue by saquibkhan** [taye/interact.js#522]
UglifyJs on Interact v1.3.0-alpha.1  is failing.

Error:
**from UglifyJs
03:57:52 SyntaxError: Unexpected token: operator (>) [./~/interactjs/src/utils/isWindow.js:1,0]**

**Comment by saquibkhan** [comment]
@taye Can you please look into this issue on priority?

**Comment by taye** [comment]
What is your build setup? Can you provide demo project that reproduces this issue?

**Comment by timothyallan** [comment]
I'll see if I can do up a sample project, but really it's just any webpack setup that uses `webpack.optimize.UglifyJsPlugin`

I've got mine setup like so:
```
plugins: [
 new webpack.optimize.UglifyJsPlugin({
      mangle: {
        toplevel: true,
        eval: true,
      },
      compress: {
        warnings: false,
        screw_ie8: true,
        sequences: true,
        properties: true,
        dead_code: true,
        drop_debugger: true,
        comparisons: true,
        conditionals: true,
        evaluate: true,
        booleans: true,
        loops: true,
        unused: true,
        hoist_funs: true,
        if_return: true,
        join_vars: true,
        cascade: true,
        negate_iife: true,
        drop_console: true,
      },
      comments: false,
      sourceMap: false,
      minimize: true,
    }),
```

**Comment by saquibkhan** [comment]
@taye We have normal webpack setup with ```webpack.optimize.UglifyJsPlugin```

You can also reproduce this if you just run UglifyJS on any file

**Another workaround Steps to repro:**
1. npm install uglify-js -g
2. cd interact.js/src/utils
3. uglifyjs isWindow.js

**Comment by taye** [comment]
I've published `v1.3.0-alpha.2` to npm. The main file is now set to the babelified `dist/interact.js`.

**Comment by saquibkhan** [comment]
@taye Thank you, i verified and its fixed now.
